BuildingSync.xsd

```diff
@@ -1192,7 +1192,7 @@
                             <xs:sequence>
                               <xs:element name="CeilingID">
                                 <xs:annotation>
-                                  <xs:documentation>ID number of the roof/ceiling type associated with this section.</xs:documentation>
+                                  <xs:documentation>ID number of the roof/ceiling type associated with this section. Valid reference paths: BuildingSync/Facilities/Facility/Systems/CeilingSystems/CeilingSystem.</xs:documentation>
```

Comment:
Any reason not to call this one `CeilingSystemID`?

Reply:
I remember we decided to keep as few breaking changes as possible, except for adding/moving `ID` in the names.

Reply:
makes sense, just wanted to make sure.